pod-master/podmaster.go

```diff
@@ -35,14 +35,15 @@ import (
 )
 
 type config struct {
-	etcdServers string
-	key         string
-	whoami      string
-	ttl         uint64
-	src         string
-	dest        string
-	sleep       time.Duration
-	lastLease   time.Time
+	etcdServers    string
+	etcdConfigFile string
```

Comment:
Ok, If you all agree I'll close this PR.

Whatever you guys decide it's ok but having a tls capable podmaster is absolutely necessary :-)